app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -205,8 +205,8 @@ class MainViewModel @Inject constructor(
     //region Current Mailbox
     private val _currentMailboxObjectId = MutableStateFlow<String?>(null)
 
-    val currentMailbox = _currentMailboxObjectId.mapLatest {
-        it?.let(mailboxController::getMailbox)
+    val currentMailbox = _currentMailboxObjectId.mapLatest { id ->
+        id?.let { mailboxController.getMailbox(it) }
```

Comment:
Revert this, we prefer the previous way of writing it.

Reply:
Doesn't work